ros_comm/eab0d3c/eab0d3c.bug

```diff
@@ -53,4 +53,4 @@ fault-codes:
   - PROGRAMMING:BROKEN-CONTRACT
 failure-codes:
   - SOFTWARE:NETWORK
-  - SYSTEM:LOSS-OF-CONTROL
+  - SYSTEM:UNRESPONSIVE
```

Comment:
This needs `LOSS-OF-FUNCTIONALITY` in addition to `UNRESPONSIVE`.

Reply:
> If ROS_HOSTNAME is set, minimal.launch does not work.

OK I think I get what you mean...